src/components/App/App.js

```diff
@@ -29,6 +45,8 @@ class App extends Component {
         <nav>
           <h1>Rancid Tomatillos</h1>
         </nav>
+        {(!this.state.movies.length && !this.state.errorMessage) && <Loading /> }
```

Comment:
Good use of conditional rendering here. Is there a way we can combine lines 48-49?